readme.md

```diff
@@ -57,6 +57,8 @@ Type: `Object`
 
 Default config.
 
+If `key` already exists in config file the value doesn't change. If the key or the config file don't yet exist they will be created and set to the default value.
```

Comment:
I don't mind documenting this, but I think it could be written more succinctly.

Reply:
I really don't mind if you have a better sentence in mind. It thought writing a pull request is as good as writing an issue :) It does sound a bit awkward...

Reply:
How about "Creates and sets the default values for `key` and writes them to disk unless they already exist."
BTW you have an awesome amount of cool repos. Just noticed :)